src/main/java/de/drachir000/utils/config/SimpleConfigLib.java

```diff
@@ -42,4 +44,47 @@ public static Configuration buildConfiguration(String source) throws JSONExcepti
 		return new Configuration(new JSONObject(source));
 	}
 	
+	/**
+	 * Saves a {@link Configuration} to a {@link File}.
+	 *
+	 * @param configuration the {@link Configuration} to be saved
+	 * @param file          the {@link File} to save the {@link Configuration} to
+	 * @throws IOException if an I/O error occurs while writing to the file
+	 * @since 1.3
+	 */
+	public static void save(Configuration configuration, File file) throws IOException {
+		
+		BufferedWriter writer = new BufferedWriter(new FileWriter(file));
+		writer.write(configuration.toString());
```

Comment:
This still only uses the old `toString()` Method, not the one, which encodes unknown object types.

Reply:
Maybe don't change this, but add a duplicate with a boolean parameter instead.

Reply:
Fixed in https://github.com/Drachir000/SimpleConfigLib/commit/f21876ca08edf3555e05d891f2ef3264c8166d28